doc/api/test.md

```diff
@@ -1393,6 +1500,419 @@ added:
+  context.mock.timers.tick(9999);
+
+  assert.strictEqual(fn.mock.callCount(), 1);
+});
+```
+
+Alternativelly, the `.tick` function can be called many times
+
+```mjs
+import assert from 'node:assert';
+import { test } from 'node:test';
+
+test('mocks setTimeout to be executed synchronously without having to actually wait for it', (context) => {
+  const fn = context.mock.fn();
+  context.mock.timers.enable(['setTimeout']);
+  const nineSecs = 9000;
+  setTimeout(fn, nineSecs);
+
+  const twoSeconds = 3000;
+  context.mock.timers.tick(twoSeconds);
+  context.mock.timers.tick(twoSeconds);
+  context.mock.timers.tick(twoSeconds);
+
+  assert.strictEqual(fn.mock.callCount(), 1);
+});
+```
+
+```js
+const assert = require('node:assert');
+const { test } = require('node:test');
+
+test('mocks setTimeout to be executed synchronously without having to actually wait for it', (context) => {
+  const fn = context.mock.fn();
+  context.mock.timers.enable(['setTimeout']);
+  const nineSecs = 9000;
+  setTimeout(fn, nineSecs);
+
+  const twoSeconds = 3000;
+  context.mock.timers.tick(twoSeconds);
+  context.mock.timers.tick(twoSeconds);
+  context.mock.timers.tick(twoSeconds);
+
+  assert.strictEqual(fn.mock.callCount(), 1);
+});
+```
+
+#### Using Clear functions
+
+As mentioned, all clear functions from timers (clearTimeout and clearInterval)
+are implicity mocked.Take a look at this example using `setTimeout`:
+
+```mjs
+import assert from 'node:assert';
+import { test } from 'node:test';
+
+test('mocks setTimeout to be executed synchronously without having to actually wait for it', (context) => {
+  const fn = context.mock.fn();
+
+  // Optionally choose what to mock
+  context.mock.timers.enable(['setTimeout']);
+  const id = setTimeout(fn, 9999);
+
+  // Implicity mocked as well
+  clearTimeout(id);
+  context.mock.timers.tick(9999);
+
+  // As that setTimeout was cleared the mock function will never be called
+  assert.strictEqual(fn.mock.callCount(), 0);
+});
+```
+
+```js
+const assert = require('node:assert');
+const { test } = require('node:test');
+
+test('mocks setTimeout to be executed synchronously without having to actually wait for it', (context) => {
+  const fn = context.mock.fn();
+
+  // Optionally choose what to mock
+  context.mock.timers.enable(['setTimeout']);
+  const id = setTimeout(fn, 9999);
+
+  // Implicity mocked as well
+  clearTimeout(id);
+  context.mock.timers.tick(9999);
+
+  // As that setTimeout was cleared the mock function will never be called
+  assert.strictEqual(fn.mock.callCount(), 0);
+});
+```
+
+#### Working with Node.js timers modules
+
+Once you enable mocking timers, [node:timers](./timers.md),
+[node:timers/promises](./timers.md#timers-promises-api) modules,
+and timers from the Node.js global context are enabled:
+
+```mjs
+import assert from 'node:assert';
+import { test } from 'node:test';
+import nodeTimers from 'node:timers';
+import nodeTimersPromises from 'node:timers/promises';
```

Comment:
I assume this simplistic approach will suddenly fail when destructuring is used? If so, there should be warnings in the docs.

Reply:
Yes! You meat adding just a Note on the docs?

Reply:
just added it. Lmk if it's what you meant. https://github.com/nodejs/node/commit/0424bf5ebaa809a98a2bcf4833dae66e5be7f43f